src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/AuthorizationFunctionalTest.java

```diff
@@ -111,6 +119,40 @@ protected String activateOrganisation(Map<String, Object> organisationCreationRe
         return organisationIdentifier;
     }
 
+    public RequestSpecification generateBearerTokenForPuiManager() {
+        Map<String, Object> response = professionalApiClient.createOrganisation();
+        log.info("RESPONSE:::::" + response);

```

Comment:
remove all newly added loggers for testing, since it will print all info for each test case.